core/src/main/java/org/infinispan/remoting/transport/jgroups/CommandAwareRpcDispatcher.java

```diff
@@ -55,6 +56,7 @@
    private static final Log log = LogFactory.getLog(CommandAwareRpcDispatcher.class);
    private static final boolean trace = log.isTraceEnabled();
    private static final boolean FORCE_MCAST = Boolean.getBoolean("infinispan.unsafe.force_multicast");
+   private static final long STAGGER_DELAY_NANOS = TimeUnit.MILLISECONDS.toNanos(Long.getLong("infinispan.stagger.delay", 5));
```

Comment:
I also found `org.infinispan.query.indexmanager.LockAcquiringBackend.MAX_QUEUE_SIZE`, used by the query module.

But this seems too much to put in this PR, I've created https://issues.jboss.org/browse/ISPN-6094